structlog_sentry/__init__.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+import sys
+
+from sentry_sdk import capture_exception, capture_message
+
+
+class SentryProcessor:
+    """Sentry processor for structlog. Uses Sentry SDK to capture events in Sentry."""
+
+    def __init__(self, sentry_log_level=logging.ERROR):
```

Comment:
I would go for `WARNING` as default. When you log something as warning, you want to know about it.